docs/sources/tempo/operations/troubleshooting/metrics-generator.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Troubleshoot metrics-generator
+menuTitle: Metrics-generator
+description: Gain an understanding of how to debug metrics quality issues.
+weight: 500
+---
+
+# Troubleshoot metrics-generator
+
+If you are concerned with data quality issues in the metrics-generator, we'd first recommend:
+
+- Reviewing your telemetry pipeline to determine the number of dropped spans. We are only looking for major issues here.
+- Reviewing the [service graph documentation]({{< relref "../../metrics-generator/service_graphs" >}}) to understand how they are built.
+
+If everything seems ok from these two perspectives, consider the following topics to help resolve general issues with all metrics and span metrics specifically.
+
+## All metrics
+
+### Dropped spans in the distributor
+
+The distributor has a queue of outgoing spans to the metrics-generators. If that queue is full then the distributor
+will drop spans before they reach the generator. Use the following metric to determine if that is happening:
+
+```
+sum(rate(tempo_distributor_queue_pushes_failures_total{}[1m]))
+```
+
+### Failed pushes to the generator
+
+For any number of reasons, the distributor can fail a push to the generators. Use the following metric to
+determine if that is happening:
+
+```
+sum(rate(tempo_distributor_metrics_generator_pushes_failures_total{}[1m]))
+```
+
+### Discarded spans in the generator
+
+Spans are rejected from being considered by the metrics-generator by a configurable slack time as well as due to user
+configurable filters. You can see the number of spans rejected by reason using this metric:
+
+```
+sum(rate(tempo_metrics_generator_spans_discarded_total{}[1m])) by (reason)
+```
+
+If a lot of spans are dropped in the metrics-generator due to your filters, you will need to adjust them. If spans are dropped
+due to the ingestion slack time, consider adjusting this setting:
+
+```
+metrics_generator:
+  metrics_ingestion_time_range_slack: 30s
+```
+
+### Max active series
+
+The generator protects itself and your remote-write target by having a maximum number of series the generator produces.  
+Use the `sum` below to determine if series are being dropped due to this limit:
+
+```
+sum(rate(tempo_metrics_generator_registry_series_limited_total{}[1m]))
+```
+
+Use the following setting to update the limit:
+
+```
+overrides:
+  metrics_generator_max_active_series: 0
+```
```

Comment:
I'd add a note here saying that the total active series will be equal to max_active_series x #replicas and that scaling up the generators will increase this total.
I know this is a troubleshooting page and this isn't strictly something that would break metrics generation, but I think it's important to understand what this knob does.